setup.cfg

```diff
@@ -56,6 +56,7 @@ doc=
         docutils==0.16  # higher versions trigger build bugs with the RTD theme
 speculos=
         speculos>=0.2.8
```

Comment:
Not really related to this PR, but would be great to bump this speculos version and to remove all references to pytesseract

Reply:
Same below in `all_backends` part

Reply:
Indeed, will clean this in another PR soon.